Containerfile.rag

```diff
@@ -0,0 +1,37 @@
+# vim: set filetype=dockerfile
+
+ARG OLS_VERSION=latest
+FROM quay.io/openshift/lightspeed-service-api:${OLS_VERSION}
+
+ARG RAG_INDEX="https://github.com/ilan-pinto/lightspeed-rag-documents/releases/latest/download/local.zip"
+ARG APP_ROOT=/app-root
+USER 0
+
+RUN microdnf install -y unzip wget \
+  && mkdir -p vector-db \
+	&& cd vector-db \
+  && wget $RAG_INDEX \
+	&& unzip local.zip \
+	&& mv local ocp-product-docs \
+	&& rm -f local.zip \
+	&& microdnf remove -y unzip wget
+
+RUN chown -R 1001:0 ${APP_ROOT} && \
+    chmod -R g+rx ${APP_ROOT}
+
+# activate virtualenv with workaround RHEL/CentOS 8+
+ENV BASH_ENV="${APP_ROOT}/venv/bin/activate" \
+    ENV="${APP_ROOT}/venv/bin/activate" \
+    PROMPT_COMMAND=". ${APP_ROOT}/venv/bin/activate" \
+    PATH="${APP_ROOT}/venv/bin:${PATH}"
+
+# default user for Python app
+USER 1001
+
+# Run the application
+EXPOSE 8080
+CMD ["uvicorn", "ols.app.main:app", "--host", "0.0.0.0", "--port", "8080"]
```

Comment:
like ENV, i would expect this to be inherited from the base image you're extending:

https://www.reddit.com/r/docker/comments/sp4q7c/do_dockerfiles_inherit_their_parent_images/

Reply:
Probably, but removing it makes it harder to look at our image and understand what it's doing. So, do you want "self documented" or do you want "inherited"? Granted the all-in-one is a hack, but the transparency is nice.

I don't have strong opinions either way, so I'm happy to defer to you.

Reply:
I want inherited because if we change how we launch OLS, i don't want this image to break or need its dockerfile updated to re-align.  This image should only override/specify things that differ.

Reply:
Good points.